deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -106,7 +106,10 @@ def build_graph(self):
         self.labels_fd.append(label)
         cost = L2Loss(in_layers=[label, regression])
         costs.append(cost)
-    all_cost = Concat(in_layers=costs, axis=1)
+    if self.mode == "classification":
+      all_cost = Concat(in_layers=costs, axis=1)
+    elif self.mode == "regression":
+      all_cost = Stack(in_layers=costs, axis=1)
```

Comment:
Turns out `L2Loss` will reduce `cost` into 1d-array. In case of multitask regression(such as qm8), this `Concat` operation will raise error, replace it with `Stack` operation